template/bash_profile.template.bash

```diff
@@ -32,6 +32,12 @@ export TODO="t"
 
 # Set this to false to turn off version control status checking within the prompt for all themes
 export SCM_CHECK=true
+# when using Git, consider installing https://github.com/romkatv/gitstatus for faster status prompt
```

Comment:
Since this is a significant new feature, please consider adding some documentation about this to the main `README.md` file. It would be good to have information about the dependency, its features, and its configuration in a central location.

Reply:
Good point, will do.

Reply:
done in #1622